globals.go

```diff
@@ -415,6 +415,7 @@ var (
 	// they're just intents, so this list omits that value.
 	ActiveStatus = []string{
 		HostRunning,
+		HostBuilding,
 		HostStarting,
 		HostProvisioning,
 		HostProvisionFailed,
```

Comment:
Do we want hosts with `HostProvisionFailed` status included?

Reply:
I think so. `HostProvisionFailed` can indicate a pretty wide range of things, but what it guarantees is that Evergreen has tried and failed to provision the host. Hosts in this state will eventually be terminated, so there should only ever be a small number of them.